testgrid/config/config_test.go

```diff
@@ -104,6 +119,22 @@ func TestConfig(t *testing.T) {
 			t.Errorf("Dashboard %v: - Must have a name", dashboardidx)
 		}
 
+		found := false
+		for _, kind := range prefixes {
+			for _, prefix := range kind {
+				if strings.HasPrefix(dashboard.Name, prefix+"-") || dashboard.Name == prefix {
+					found = true
+					break
+				}
+			}
+			if found {
+				break
+			}
+		}
```

Comment:
the prefix check probably can be in it's own function, as dashboard and dashboardGroup shares the same logic

Reply:
Meh